programmers/난이도별/level02.오픈채팅방/Leemainsw.js

```diff
@@ -0,0 +1,32 @@
+function solution(record) {
+    let nickNames = [];
+    let printUsers = [];
+    
+    const Enter = '님이 들어왔습니다.';
+    const Leave = '님이 나갔습니다.';
+    
+    const printString = {Leave, Enter};
```

Comment:
```suggestion
    const printString = {
        Leave: '님이 나갔습니다.',
        Enter: '님이 들어왔습니다.'
    };
```
`Leave`와 `Enter`가 printString 안에서만 쓰인다면 한 번에 선언하는 게 더 좋을 것 같아요.